index.js

```diff
@@ -55,6 +55,13 @@ function read(filename) {
   return fs.readFileSync(filename, 'utf8');
 }
 
+function requireJsonOrFallback(filename, fallback) {
+  if (fs.existsSync(filename + '.json')) {
+    return require(filename);
+  }
+  return fallback;
```

Comment:
To my future self and others:

`existsSync` is deprecated, but a simple `try { require(filename); } catch (e) {}` wouldn't work as well since we want to report if the JSON is ill-formed.

To @RobLoach, you might want to use something like `(fs.accessSync || fs.existsSync)(filename + '.json')` to make sure the code still works when/if Node.js/io.js removes `existsSync`.